cylc/flow/network/server.py

```diff
@@ -291,6 +310,24 @@ def api(self, endpoint=None):
             return '%s\n%s' % (head, tail)
         return 'No method by name "%s"' % endpoint
 
+    @authorise(Priv.CONTROL)
+    @expose
+    def broadcast(
+            self,
+            mode,
+            cycle_points=None,
+            tasks=None,
+            settings=None
+    ):
+        """Put or clear broadcasts."""
+        if mode == 'put_broadcast':
+            return self.schd.task_events_mgr.broadcast_mgr.put_broadcast(
+                cycle_points, tasks, settings)
+        if mode == 'clear_broadcast':
+            return self.schd.task_events_mgr.broadcast_mgr.clear_broadcast(
+                cycle_points, tasks, settings)
+        raise ValueError('TEMP: TODO: FixMe!')
```

Comment:
There's a `raise ValueError('Unknown command...')` in the same file... maybe we can use a similar message here?